torch_geometric/loader/neighbor_loader.py

```diff
@@ -494,3 +520,34 @@ def to_index(tensor):
                     remote_backend_utils.num_nodes(feature_store, graph_store,
                                                    input_nodes))
             return node_type, to_index(input_nodes)
+
+        assert isinstance(input_nodes, TensorAttr)
+        assert input_nodes.is_set('attr_name')
+
+        node_type = getattr(input_nodes, 'group_name', None)
+        if not input_nodes.is_set('index') or input_nodes.index is None:
+            num_nodes = feature_store.get_tensor_size(input_nodes)[0]
+            return node_type, range(num_nodes)
+        return node_type, input_nodes.index
+
+
+def get_mixed_sampling_nodes(data: HeteroData,
+                             input_nodes: List[InputNodes]) -> SamplingNodes:
```

Comment:
Document or perhaps rename? `mixed` isn't super clear to me (this is just getting sampling nodes for different node types, right?)

Reply:
Yes good point, and I think you're right we could merge these two functions. I originally spit them because it was being used in the sampler init, but I've removed this. Let me clean it up.